.pipelines/templates/load-test.yml

```diff
@@ -27,48 +27,45 @@ jobs:
       - name: OPERATION_MODE
         value: ${{ format('{0}', operationMode) }}
       steps:
-        - task: GoTool@0
-          inputs:
-            version: 1.15
-
         - bash: |
             git clone -b $(checkout.branch) $(checkout.repo) $(perf-tests.repo.path)
           displayName: Checkout $(checkout.repo) @ $(checkout.branch)
 
-        - template: build-images.yml
+        - template: az-login.yml
+          parameters:
+            subscription_id: $(LOAD_TEST_SUBSCRIPTION_ID)
 
-        - script: |
-            az account set -s=$(LOAD_TEST_SUBSCRIPTION_ID)
-          displayName: "Set subscription for load tests"
+        - template: install-helm.yml
+
+        - template: build-images.yml
```

Comment:
The images are built and pushed to the registry in the CI sub. The subscription used for deploying the cluster is the load test sub. This would cause the build images to fail because the sub is set to load test sub in the previous step. The image push will fail here?

Reply:
I believe it won't fail here because I granted the agent access to both subscriptions and `az acr login -n $REGISTRY` should succeed. Let me verify it again.